Source/WTF/wtf/text/AtomString.h

```diff
@@ -62,7 +62,7 @@ class AtomString final {
     String releaseString() { return WTFMove(m_string); }
 
     // FIXME: What guarantees this isn't a SymbolImpl rather than an AtomStringImpl?
-    AtomStringImpl* impl() const { return static_cast<AtomStringImpl*>(m_string.impl()); }
+    AtomStringImpl* impl() const { SUPPRESS_MEMORY_UNSAFE_CAST return static_cast<AtomStringImpl*>(m_string.impl()); }
```

Comment:
Isn't that a good example of something that should be a downcast? Even the FIXME above makes it scary.

Reply:
Ok, we can revert this part.